src/axom/mint/docs/sphinx/sections/citations.rst

```diff
@@ -0,0 +1,23 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+
+.. _LLNL: https://www.llnl.gov/
+.. _Umpire: https://umpire.readthedocs.io/en/develop/
+.. _Mint Doxygen API Documentation: https://lc.llnl.gov/axom/docs/doxygen/axom_doxygen/html/classaxom_1_1mint_1_1Mesh.html
+.. _Axom Quick Start Guide: https://axom.readthedocs.io/en/develop/docs/sphinx/quickstart_guide/index.html
+.. _Conduit: https://llnl-conduit.readthedocs.io/en/latest/index.html
+.. _Blueprint: https://llnl-conduit.readthedocs.io/en/latest/blueprint.html
+.. _RAJA: https://raja.readthedocs.io/en/master/index.html
+.. _Axom Toolkit: https://axom.readthedocs.io/en/develop/
```

Comment:
Yes -- we need to add labels to the other sections and use that instead. Let's do that in another PR since this change should spans across components.